components/jaeger-agent.libsonnet

```diff
@@ -15,6 +15,8 @@ local containerEnv = container.envType;
       containerEnv.fromFieldPath('NAMESPACE', 'metadata.namespace'),
       containerEnv.fromFieldPath('POD', 'metadata.name'),
     ]) +
+    container.mixin.resources.withRequests({ cpu: '32m', memory: '16Mi' }) +
```

Comment:
Judging by current usage these would be more applicable:

24 Mi memory request
10m CPU request